### PR TITLE
Reorganize package.json scripts

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

**Comment by jbuck** [comment]
@gvn r?

The general idea here was:

* put most-used scripts at the top (start and test)
* group and sort everything else somewhat alphabetically

File: package.json [hunk 1]
```diff
@@ -44,21 +51,24 @@
...
     "hoek": "^2.11.1",
     "hyperquest": "^1.0.1",
+    "imports-loader": "^0.6.3",
     "joi": "^6.0.8",
+    "jsx-loader": "^0.12.2",
     "react": "^0.12.2",
     "react-router": "^0.12.4",
+    "react-router-stub": "0.0.5",
     "react-validation-mixin": "^4.0.3",
+    "should": "^5.2.0",
```
Comment (gvn) [inline package.json]: Dev dep?

File: package.json [hunk 2]
```diff
@@ -4,19 +4,26 @@
...
-    "build": "npm run build:testassets && npm run build:js",
-    "watch-css": "autoless --source-map templates/less public",
-    "watch-js": "npm run build:js -- --watch",
-    "jshint": "jshint  lib test web *.js",
+
     "jscs": "jscs lib test web *.js",
-    "js-beautify": "js-beautify  lib test web *.js -r ",
-    "validate": "npm run jshint && npm run jscs"
+    "jshint": "jshint lib test web *.js",
+    "js-beautify": "js-beautify lib test web *.js -r",
```
Comment (gvn) [inline package.json]: Maybe `jsbeautify` instead? I dunno...

**Comment by gvn** [comment]
LGTM...just a few comments that I'll leave up to you, but R+!

File: package.json (hunk 2 above)
Comment (jbuck) [inline package.json]: will fix

File: package.json (hunk 1 above)
Comment (jbuck) [inline package.json]: sadly not, since the build step *also* builds our JS files it's actually a `build:js` dependency

**Pull request merged**